js/gallery.js

```diff
@@ -4,23 +4,22 @@ const renderPhotos = (photos) =>{
   const photoTemplate = document.querySelector('#picture').content;
   const photosList = document.querySelector('.pictures');
 
-  const createPhotos = photos; // массив фотографий
+  const createPhotos = photos;
 
-  const createListPictures = document.createDocumentFragment();
+  const createListPhotos = document.createDocumentFragment();
 
 
-  // заполняет созданный элемент
   createPhotos.forEach(({url, description, likes, comments}) => {
     const element = photoTemplate.cloneNode(true);
-    const pictureImg = element.querySelector('.picture__img');
-    pictureImg.src = url;
-    pictureImg.alt = description;
+    const photoImg = element.querySelector('.picture__img');
```

Comment:
Я бы ко всем селекторам элеента добавлял название что это элемент. Сам с таким не работаю, но вроде это логично
```suggestion
    const photoImgElement = element.querySelector('.picture__img');
```